.prettierrc.js

```diff
@@ -1,3 +1,6 @@
-module.exports = {
+(module.exports = {
   ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
-};
+}),
+  (module.exports.importOrder = ['^components/(.*)$', '^[./]']),
+  (module.exports.importOrderSeparation = true),
+  (module.exports.importOrderSortSpecifiers = true);
```

Comment:
`module.exports` is a normal variable, there is no need for the parens here. Should look like this:
```
module.exports = {
  ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
};
module.exports.importOrder = ['^components/(.*)$', '^[./]'];
module.exports.importOrderSeparation = true;
module.exports.importOrderSortSpecifiers = true;
```

Reply:
It appears that by running `yarn prettier --write ./` from the main folder, also the configuration file itself is modified. Those parentheses are due to the `prettier` formatting.

The reason is that after each variable declaration, typescript expects a semicolon, not a comma. By introducing semicolons, the issue is fixed.

This does not affect the format of all other files.

Reply:
Looks great!